app/src/main/java/com/example/jetpackcompose/HomeScreen.kt

```diff
@@ -0,0 +1,172 @@
+            items(tasks) {
+                TaskItem(
+                    task = it,
+                    modifier = Modifier.padding(dimensionResource(R.dimen.padding_small))
+                )
+            }
+        }
+    }
+}
+
+/**
+ * Composable that displays a list item containing a dog icon and their information.
+ *
+ * @param task contains the data that populates the list item
+ * @param modifier modifiers to set to this composable
+ */
+@Composable
+fun TaskItem(
+    task: Task,
+    modifier: Modifier = Modifier
+) {
+    Card(
+        modifier = modifier
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(dimensionResource(R.dimen.padding_small))
+        ) {
+            Spacer(modifier = Modifier.width(50.dp))
+            TaskInformation(task.id, task.name, task.createdDate)
+        }
+    }
+}
+
+
+
+/**
+ * Composable that displays a Top Bar with an icon and text.
+ *
+ * @param modifier modifiers to set to this composable
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun WoofTopAppBar(modifier: Modifier = Modifier) {
+    CenterAlignedTopAppBar(
+        title = {
+            Row(
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Image(
+                    modifier = Modifier
+                        .size(88.dp)
+                        .padding(8.dp),
+                    painter = painterResource(R.drawable.ic_app),
+                    contentDescription = null)
+                Text(
+                    text = "Life List - Tasks"
+                )
+            }
+        },
+        modifier = modifier
+    )
+}
+
+
+
+
+@Composable
+fun TaskInformation(
+    taskId: Int,
+    @StringRes taskName: Int,
+    @StringRes createdDate: Int,
+    modifier: Modifier = Modifier
+) {
+    Column(modifier = modifier) {
+        Text(
+            text = stringResource(R.string.tasksd, taskId),
+            style = MaterialTheme.typography.displayMedium,
+            modifier = Modifier.padding(top = dimensionResource(id = R.dimen.padding_small))
+        )
+        Text(
+            text = stringResource(taskName),
+            style = MaterialTheme.typography.bodyLarge
+        )
+        Row(
+            modifier = Modifier.padding(top = dimensionResource(id = R.dimen.padding_small)),
+            verticalAlignment = Alignment.CenterVertically
+        ){
+            Text(
+                text = stringResource(createdDate),
+                style = MaterialTheme.typography.displayMedium
+            )
+            Spacer(modifier = Modifier.width(50.dp))
+
+            Button(
+                onClick = {}
+
+            ){
+                Text(text = "Done")
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+fun HomeScreenPreview() {
+    JetpackComposeTheme(darkTheme = false) {
+        WoofApp()
```

Comment:
I'm confused here, the app was supposed to have one activity (one tk file) but I'm finding elements in here that are noisy. Please remove all elements that are not being used / executed.